src/utils/utils.py

```diff
@@ -48,11 +49,35 @@ def wrap_function(*args, **kwargs):
     return wrap_function
 
 
-def error(message):
-    logging.error("")
-    logging.error("[ERROR]: " + message)
-    logging.error("")
-    sys.exit(1)
+def console_error(*argv, exit=True):
+    if len(argv) > 1:
+        logging.error("ERROR: " + f"[{argv[0]}] {argv[1]}")
+    else:
+        logging.error("ERROR: " + f"{argv[0]}")
+    if exit:
+        sys.exit(1)
+
+
+def console_log(*argv, indent_level=0):
+    indent = ""
+    if indent_level >= 1:
+        indent = " " * 3 * indent_level + "|-> "  # spaces per indent level
+
+    if len(argv) > 1:
+        logging.info(indent + f"[{argv[0]}] {argv[1]}")
+    else:
+        logging.info(indent + f"{argv[0]}")
+
+
+def console_debug(*argv):
+    if len(argv) > 1:
+        logging.debug(f"[{argv[0]}] {argv[1]}")
+    else:
+        logging.debug(f"{argv[0]}")
+
+
+def console_warning(msg: str):
+    logging.warning(msg)
```

Comment:
Following format of all other log helpers to support (optional) second parameter
```suggestion
def console_warning(msg: str):
    if len(argv) > 1:
        logging.warning(f"[{argv[0]}] {argv[1]}")
    else:
        logging.warning(f"{argv[0]}")
```